src/main/java/com/revature/domain/QuestionComment.java

```diff
@@ -0,0 +1,101 @@
+package com.revature.domain;
+
+import javax.persistence.*;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import java.io.Serializable;
+
+@Entity
+@Table(name="ie_question_comment")
+public class QuestionComment implements Serializable {
+
+	@Id
+	@Column(name="qc_id")
+	@GeneratedValue(strategy=GenerationType.SEQUENCE, generator="questionCommentSeq")
+	@SequenceGenerator(allocationSize=1, name="questionCommentSeq", sequenceName="question_comment_seq")
+	private Integer id;
+	
+	@Column(name="qc_comment_text")
+	private String commentText;
+	
+	@ManyToOne(fetch=FetchType.EAGER)
+	@JoinColumn(name="qc_eid")
+	private QuestionEval questionEval;
+
+	public QuestionComment(){}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1186)